raiden-network/light-client
# dApp: Displays notifications

Fixes #1758
Fixes #1374  

**Short description**
This PR adds frontend logic and templates for receiving, viewing, and deleting notifications in the notification panel. A first notification to be added should be the notification regarding balance proof, issue #1374 . This needs additional work outside this PR, @kelsos is looking into that.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Go to the UDC screen
2. Withdraw an amount
3. Wait 100 blocks
4. Notification area should show a small red dot and contain a notification
5. Enable receiving/ have enough in UDC
6. Send from python client some tokens
7. Close LC
8. Close channel from PyC
9. Wait for Settlement period to pass
10. Open LC 
11. Notification area should show a small red dot and contain a notification
